sdk/textanalytics/Azure.AI.TextAnalytics/tests/TextAnalyticsClientMockTests.cs

```diff
@@ -422,6 +422,120 @@ public async Task AnalyzeSentimentMixedSentenceSentiment()
             Assert.AreEqual(TextSentiment.Mixed, response.Sentences.FirstOrDefault().Sentiment);
         }
 
+        [Test]
+        public async Task AnalyzeSentimentOpinionInOtherSentence()
+        {
+            using var stream = new MemoryStream(Encoding.UTF8.GetBytes(@"
+                {
+                    ""documents"": [
```

Comment:
The intent of the mock is to reuse the opinion of another sentence, so I'm creating a fake output for the scenario, as we are still not sure how the service will manage this.